package/batocera/core/batocera-configgen/configgen/configgen/Emulator.py

```diff
@@ -76,7 +76,7 @@ def dict_merge(dct, merge_dct):
         :param merge_dct: dct merged into dct
         :return: None
         """
-        for k, v in merge_dct.iteritems():
+        for k, v in list(merge_dct.items()):
```

Comment:
Why `list()` here? In python 3 you can just `for k,v in merge_dct.items()`

Reply:
Fixed